content/developer/reference/cli.rst

```diff
@@ -740,35 +739,6 @@ Scaffolding is available via the :command:`odoo-bin scaffold` subcommand.
 
 This will create module *my_module* in directory */addons/*.
 
-.. _reference/cmdline/populate:
-
-Database Population
-===================
-
-.. program:: odoo-bin populate
-
-Odoo CLI supports database population features. If the feature is
-:ref:`implemented on a given model <reference/performance/populate/methods>`, it allows automatic
-data generation of the model's records to test your modules in databases containing non-trivial
-amounts of records.
-
-.. code-block:: console
-
-    $ odoo-bin populate
-
-.. option:: --models
-
-    list of models for which the database should be filled
-
-.. option:: --size (small|medium|large)
-
-    population size, the actual records number depends on the model's `_populate_sizes` attribute.
-    The generated records content is specified by the :meth:`~odoo.models._populate_factories` method
-    of a given model (cf. the :file:`populate` folder of modules for further details).
-
-.. seealso::
-   :ref:`reference/performance/populate`
-
```

Comment:
Since you're replacing and not dropping the `populate` feature, you could at least document it instead of simply removing the whole section.